index.html

```diff
@@ -113,13 +113,13 @@
     //
     //-------------------------------------------------------------------------
 
```

Comment:
Could we put a comment title here just to show this is the 'block colour' section..?